index.html

```diff
@@ -609,6 +609,77 @@ <h3>Ed25519Signature2018</h3>
           </pre>
       </section>
 
+      <section id="JsonWebSignature2020">
+
+        <h3>JsonWebSignature2020</h3>
+          <p>
+A Linked Data signature is used for digital signatures on RDF Datasets.
+The default canonicalization mechanism is specified in the RDF Dataset
+Normalization specification, which deterministically names all
+unnamed nodes. The default signature mechanism
+uses a SHA-256 digest and JWS to perform the digital signature.
+          </p>
+          <dl>
+            <dt>Status</dt>
+            <dd property="vs:term_status">stable</dd>
+            <dt>Parent Class</dt>
+            <dd>Signature</dd>
+            <dt>Expected properties</dt>
+            <dd>verificationMethod, jws</dd>
+            <dt>Signature Properties</dt>
+            <dd>
+              <dl>
+                <dt>Default Canonicalization Algorithm</dt>
+                <dd>
+<a rel="canonicalizationAlgorithm"
+  href="https://w3id.org/rdf#URDNA2015">
+https://w3id.org/rdf#URDNA2015</a>
+                </dd>
+                <dt>Default Signature Algorithm</dt>
+                <dd>
+<a rel="signatureAlgorithm"
+  href="https://tools.ietf.org/html/rfc8037#section-3.1">
+  https://tools.ietf.org/html/rfc8037#section-3.1</a>
+                </dd>
+              </dl>
+            </dd>
+          </dl>
+          <p>
+            The JWS used with Linked Data Signatures is the detached and unencoded payload variant. See the links below for details:
+          </p>
+          <ul>
+            <li>
+            <a href="https://tools.ietf.org/html/rfc7797">JSON Web Signature (JWS) Unencoded Payload Option</a>
+          </li>
+          <li>
+            <a href="https://medium.com/swlh/json-web-signature-jws-and-jws-detached-for-a-five-year-old-88729b7b1a68">JSON Web Signature (JWS) and JWS Detached for a five-year-old.</a>
```

Comment:
I see this is also in Ed25519Signature2018.  Links to actual specs would be better than links to posts for 5 year olds.

Reply:
The link to the spec is above.... I have yet to encounter a single developer who was not confused by rfc7797... its obscure in the JOSE ecosystem, if you have a better friendly explainer, I'd be in favor of adding another link / replacing this one...